src/configurations/cancercomplexity/synapseConfigs/tools.ts

```diff
@@ -15,11 +15,13 @@ export const toolsConfiguration: CardConfiguration = {
     description: 'description',
     secondaryLabels: [
       'publicationTitle',
-      'inputDataType',
-      'outputDataType',
-      'softwareLanguage',
+      'inputData',
+      'inputFormat',
+      'outputData',
+      'outputFormat',
+      'language',
       'toolType',
-      'grantName',
+      'grantNumber',
       'consortium',
     ],
     link: 'homepageUrl',
```

Comment:
Seems like the new column with this data is now named `homepage`

Reply:
Nice catch!  Pushing a fix.